Components/Queues/Drivers/StompQueueDriver.php

```diff
@@ -93,11 +117,33 @@ public function getStompVersion()
     }
 
     /**
-     * @param string $stompVersion
+     * Set the version to Stomp driver.
+     *
+     * @param string $version
+     */
+    public function setStompVersion(string $version = self::STOMP_VERSION)
+    {
+        $this->stompVersion = $version;
+    }
+
+    /**
+     * Set the timeout to Stomp driver.
+     *
+     * @param int $connectionTimeout
      */
-    public function setStompVersion($stompVersion)
+    public function setTimeout(int $connectionTimeout = 3)
```

Comment:
I would rename this `setTimeout()` to `setConnectionTimeout()` to avoid the missleading

Reply:
Done.